codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/ActionTestKitGeneratorSuite.scala

```diff
@@ -0,0 +1,218 @@
+          |import com.akkaserverless.javasdk.testkit.impl.StubActionCreationContext;
+          |import com.example.actions.CounterJournalToTopicAction;
+          |import com.example.actions.CounterTopicApi;
+          |import com.example.domain.CounterDomain;
+          |import com.google.protobuf.Any;
+          |import com.google.protobuf.Empty;
+          |import java.util.ArrayList;
+          |import java.util.List;
+          |import java.util.Optional;
+          |import java.util.function.Function;
+          |
+          |public final class CounterJournalToTopicActionTestKit {
+          |
+          |  private Function<ActionCreationContext, CounterJournalToTopicAction> actionFactory;
+          |
+          |  private CounterJournalToTopicAction createAction() {
+          |    CounterJournalToTopicAction action = actionFactory.apply(new StubActionCreationContext());
+          |    action._internalSetActionContext(Optional.of(new StubActionContext()));
+          |    return action;
+          |  };
+          |
+          |  public static CounterJournalToTopicActionTestKit of(Function<ActionCreationContext, CounterJournalToTopicAction> actionFactory) {
+          |    return new CounterJournalToTopicActionTestKit(actionFactory);
+          |  }
+          |
+          |  private CounterJournalToTopicActionTestKit(Function<ActionCreationContext, CounterJournalToTopicAction> actionFactory) {
+          |    this.actionFactory = actionFactory;
+          |  }
+          |
+          |  private <E> ActionResult<E> interpretEffects(Action.Effect<E> effect) {
+          |    return new ActionResultImpl(effect);
+          |  }
+          |
+          |  public ActionResult<CounterTopicApi.Increased> increase(CounterDomain.ValueIncreased valueIncreased) {
+          |    Action.Effect<CounterTopicApi.Increased> effect = createAction().increase(valueIncreased);
+          |    return interpretEffects(effect);
+          |  }
+          |
+          |  public ActionResult<CounterTopicApi.Decreased> decrease(CounterDomain.ValueDecreased valueDecreased) {
+          |    Action.Effect<CounterTopicApi.Decreased> effect = createAction().decrease(valueDecreased);
+          |    return interpretEffects(effect);
+          |  }
+          |
+          |  public ActionResult<Empty> ignore(Any any) {
+          |    Action.Effect<Empty> effect = createAction().ignore(any);
+          |    return interpretEffects(effect);
+          |  }
+          |
+          |}""".stripMargin
+
+    assertNoDiff(sourceCode, expected)
+  }
+
+  test("it can generate an specific Test stub for the entity") {
+
+    val sourceCode = ActionTestKitGenerator.generateTestSourceCode(service)
+
+    val expected =
+      """/* This code was generated by Akka Serverless tooling.
+        | * As long as this file exists it will not be re-generated.
+        | * You are free to make changes to this file.
+        | */
```

Comment:
Note that we did some changes wrt the generated banner last week in #550

Reply:
Thanks!!